### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by zigasinko**
Implemented landing page.

### CODE REVIEW

**Comment by hadalin** [comment]
I think @dblenkus should review this?

File: .eslintrc.js
Comment (dblenkus) [inline .eslintrc.js:47]: This does not belong in this commit.

File: src/components/app/app.test.tsx
Comment (dblenkus) [inline src/components/app/app.test.tsx:3]: Isn't it usual to use Pascal case for files with components?

File: src/components/app/app.test.tsx
Comment (dblenkus) [inline src/components/app/app.test.tsx:5]: Why have you decided to use this library for testing? Have you considered the built-in one or Enzyme, which is recommended by React? https://reactjs.org/docs/test-utils.html

File: src/components/landing/appBar/appBar.tsx
Comment (dblenkus) [inline src/components/landing/appBar/appBar.tsx]: Make `NavButton` a separate component.

File: src/components/landing/home/home.tsx
Comment (dblenkus) [inline src/components/landing/home/home.tsx]: Why do you define another `AppBar` here?

File: .eslintrc.js
Comment (zigasinko) [reply .eslintrc.js:47]: Had to be done, because I added some .png images to landing page.

File: src/components/app/app.test.tsx
Comment (zigasinko) [reply src/components/app/app.test.tsx:3]: Since we'r using different OSes, the recommended approach is everything should start with a lower case. Components are in all files that end with ".tsx".

File: src/components/app/app.test.tsx
Comment (zigasinko) [reply src/components/app/app.test.tsx:5]: It ships by default with create-react-app. Didn't really start working on tests / testing framework. Will look into in when we finalize a few things (store, ui library).

File: src/components/landing/home/home.tsx
Comment (zigasinko) [reply src/components/landing/home/home.tsx]: Typo..

File: src/components/landing/appBar/appBar.tsx
Comment (zigasinko) [reply src/components/landing/appBar/appBar.tsx]: Done.

File: src/components/app/app.test.tsx
Comment (dblenkus) [reply src/components/app/app.test.tsx:3]: Who recommends this? What is the reasoning behind it?

File: package.json
Comment (dblenkus) [inline package.json]: This is probably not needed as the related package is not installed.

File: package.json
Comment (dblenkus) [inline package.json:58]: It is preferred to put configuration changes into a separate commit because this one is named "Add router and landing page".

File: src/components/landing/appBar/appBar.tsx
Comment (dblenkus) [reply src/components/landing/appBar/appBar.tsx]: Where is this done? The directive repeats five times and has 15 lines of (almost) the same code.

File: src/components/app/app.test.tsx
Comment (zigasinko) [reply src/components/app/app.test.tsx:3]: Can't find the article right now, but it's a problem if someone on a case sensitive OS creates a files that already exists with different casing. Besides that changing a from/to component would be impossible on a windows / osx, because github ignores casing (core.ignorecase is true by default).
So it's either all lower case (kebab-case) or lowercamelcase.

File: src/components/landing/appBar/appBar.tsx
Comment (zigasinko) [reply src/components/landing/appBar/appBar.tsx]: NavButton also extracted to a new component.

File: src/components/app/app.test.tsx
Comment (dblenkus) [reply src/components/app/app.test.tsx:3]: I imagine that create-react-app uses reasonable defaults, so I would prefer to stick with Pascal case.
I am not convinced by your arguments because that shouldn't be a problem if git doesn't accept two files with the same name and different cases.

File: public/index.html
Comment (dblenkus) [inline public/index.html]: Remove "LOCAL"

File: public/index.html
Comment (dblenkus) [inline public/index.html:13]: Is there a reason why have you removed `manifest.json` and put favicons here?

File: src/components/landing/landingPage.tsx
Comment (dblenkus) [inline src/components/landing/landingPage.tsx]: This solution looks very fragile. It there another way of solving this?

**Review by dblenkus — COMMENTED** [review]
Code looks good in general.

Please (re)organize commits so that each contains a minimal self-contained change, e.g. ESLint configuration changes, favicons, landing page,...

File: public/index.html
Comment (zigasinko) [reply public/index.html:13]: manifest.json is not needed since dicty doesn't use any PWA functionalities. Favicons were migrated from original dictyExpress landing page.

File: src/components/app/app.test.tsx
Comment (zigasinko) [reply src/components/app/app.test.tsx:3]: Found a couple: https://www.w3schools.com/js/js_conventions.asp
https://stackoverflow.com/questions/7273316/what-is-the-javascript-filename-naming-convention
https://www.robinwieruch.de/javascript-naming-conventions

I'd rather suggest going to kebab-case instead of Pascal case. Besides the drawbacks described above and in the articles, it would take me forever to rename each file/folder with git move command (the only way to truly change filename capitalization).

File: src/components/landing/landingPage.tsx
Comment (zigasinko) [reply src/components/landing/landingPage.tsx]: I agree and it was just a temporary solution. Now I added a "window load" event and attached initial scroll (if user refreshes the page or opens a new URL with section hash).

File: src/components/app/app.test.tsx
Comment (zigasinko) [reply src/components/app/app.test.tsx:3]: We will get back to this comment after first this and first module is merged to master.

**Review by dblenkus — APPROVED** [review]

**Pull request merged**
